src/server.c

```diff
@@ -3243,7 +3243,7 @@ static void propagateNow(int dbid, robj **argv, int argc, int target) {
      * replica pause (otherwise data may be lost during a failover) */
```

Comment:
This comment doesn't mention that we can replicate DEL even when clients writes are paused. I think we should mention when and why it can happen.